tests/document_ops.rs

```diff
@@ -15,13 +15,18 @@ async fn doc_list() -> Result<(), IronOxideErr> {
     let sdk = initialize_sdk().await?;
     let (other_user, _) = init_sdk_get_user().await;
     let doc = "secret".to_string().into_bytes();
-    // grant_to_author is false, but doc should still come back in document_list
+    // grant_to_author is false, but doc should not come back in document list
```

Comment:
A nitpick, but I think it's important for clarity
```suggestion
    // grant_to_author is false, so doc should not come back in document list
```